src/main/api/opds.ts

```diff
@@ -66,14 +60,22 @@ export class OpdsApi {
 
     public async browse(data: any): Promise<THttpGetOpdsResultView> {
         const { url } = data;
-        return await httpGet(url, {}, async (opdsFeedData) => {
+        return await httpGet(url, {
+            timeout: 10000,
+        }, async (opdsFeedData) => {
             // let opds2Publication: OPDSPublication = null;
             let opds2Feed: OPDSFeed = null;
 
             if (opdsFeedData.isFailure) {
                 return opdsFeedData;
             }
 
+            debug("opdsFeed content-type", opdsFeedData.contentType);
+            if (!opdsFeedData.contentType.startsWith("application/json") &&
```

Comment:
This string comparison will fail `application/opds+json` (OPDS 2.0)
https://github.com/opds-community/drafts/blob/master/opds-2.0.md

Reply:
I add it.
We need a constant array of all contentType available in opds.

Reply:
https://github.com/readium/readium-desktop/pull/542/commits/8bbaf722c134b50626397a828dec198bdfb6d6c3